.github/workflows/refresh_voc_and_counts.yml

```diff
@@ -0,0 +1,45 @@
+name: Refresh vocabulary and counts tables
```

Comment:
@Sigfried `modified:   termhub-csets (modified content)`: Do you want me to include these updated files I downloaded in this PR?